### PR TITLE
[Feature] #14 Construct EditorMainView

### LINKED ISSUE #14: [Feature] Construct EditorMainView's CardView
# 유저는 에디터 콘텐츠를 구경할 수 있다
- 콘텐츠 카드뷰의 UI를 구성한다

## 🤔 Completion conditions

- [ ] Construct EditorContents CardView UI like the design of a sketch

### PR DESCRIPTION
**Pull request by Gobans**
# Issue Number
🔒 Close #14 

## Changes

- Changed files
campair/campair/Application/Assets.xcassets/FoldingChair.imageset/Contents.json
campair/campair/Application/Assets.xcassets/FoldingChair.imageset/FoldingChair.png
campair/campair/Application/Assets.xcassets/contentCard1.imageset/Contents.json
campair/campair/Application/Assets.xcassets/contentCard1.imageset/contentCard1.png
campair/campair/View/Editor/EditorCardView.swift
campair/campair/View/Editor/EditorEquipListView.swift
campair/campair/View/Editor/EditorMainView.swift

## New features

- MainView's cardView
- MainView's EquipmentList
image
image

## Review points

- Card View's Implement
- EquipmentList's Implement

## Questions

- Depending on device size, It changed dynamically. BUT because of changing of size, EquipmentList's cards were changed so small and border also changed too round, Is there any good idea to deal with it?

## Checklist

- [x] Is the branch you are merging on correct?
- [x] Do you comply with coding conventions?
- [x] Are there any changes not related to PR?
- [x] Has my code been self-reviewed?

### CODE REVIEW

**Comment by rriver2** [comment]
good ~~~ 👍🏻

File: campair/campair/View/Editor/EditorEquipListView.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            ForEach(0..<4) { _ in
+                VStack {
+                    Image("FoldingChair")
+                        .resizable()
+                        .scaledToFit()
+                        .padding(.horizontal, 23)
+                        .padding(.vertical, 20)
+                        .background(
+                            RoundedRectangle(cornerRadius: 30)
+                                .foregroundColor(Color(#colorLiteral(red: 0.9317113757, green: 0.8984663486, blue: 0.8601023555, alpha: 1)))
```
Comment (unnnyong) [inline campair/campair/View/Editor/EditorEquipListView.swift:22]: Color Asset 등으로 색상들은 추가될 예정인지 궁금합니다 ~~~ !
다크모드일때의 디자인에서 사용되는 색상 대비 등은 괜찮았나요 ? 👀

File: campair/campair/View/Editor/EditorCardView.swift [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            }
+            .padding(.top, 34)
+            .padding(.leading, 20)
+            Image("ContentCard1")
+                .resizable()
+                .scaledToFit()
+                .padding(20)
+        }
+        .background(
+            RoundedRectangle(cornerRadius: 30)
```
Comment (sustainable-git) [inline campair/campair/View/Editor/EditorCardView.swift:31]: how about 15

File: campair/campair/View/Editor/EditorEquipListView.swift [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        HStack(spacing: 6) {
+            ForEach(0..<4) { _ in
+                VStack {
+                    Image("FoldingChair")
+                        .resizable()
+                        .scaledToFit()
+                        .padding(.horizontal, 23)
+                        .padding(.vertical, 20)
+                        .background(
+                            RoundedRectangle(cornerRadius: 30)
```
Comment (sustainable-git) [inline campair/campair/View/Editor/EditorEquipListView.swift:21]: how about 15

File: campair/campair/View/Editor/EditorCardView.swift [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                Text("차로 떠나는 피크닉")
+                    .foregroundColor(.white)
+                    .font(.title)
+                    .fontWeight(.bold)
+            }
+            .padding(.top, 34)
+            .padding(.leading, 20)
+            Image("ContentCard1")
+                .resizable()
+                .scaledToFit()
```
Comment (sustainable-git) [inline campair/campair/View/Editor/EditorCardView.swift:27]: Dame, this code is so tight!

**Review by sustainable-git — APPROVED** [review]
GGOODD!

File: campair/campair/View/Editor/EditorEquipListView.swift (hunk 1 above)
Comment (Gobans) [reply campair/campair/View/Editor/EditorEquipListView.swift:22]: 다크모드는 나중에 고려할 사항으로 정하였습니다..! 이슈로 등록 해놓았습니다. 👍 #35

File: campair/campair/View/Editor/EditorEquipListView.swift (hunk 3 above)
Comment (Gobans) [reply campair/campair/View/Editor/EditorEquipListView.swift]: It changed 18. Thanks!

**Pull request merged**